src/Hydrators/IssueHydrator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace ITBugTracking\Hydrators;
+
+use PDO;
+use ITBugTracking\Entities\Issue;
+
+class IssueHydrator
+{
+    public static function getIssues(PDO $db, $completedFilter): array|null
+    {
+        $queryString = 'SELECT `issues`.`id`,`issues`.`title`,`issues`.`description`,`issues`.`date_created`,`issues`.`reporter`,`issues`.`department`,`comments`.`issue_id`,COUNT(`comments`.`issue_id`) AS `comment_count`,`issues`.`completed`,`severities`.`name` AS `severity`
```

Comment:
You're pulling out at least one more field than you need to, `` `comments`.`issue_id` `` is not used.

Reply:
done